src/test/java/com/eppo/sdk/dto/EppoAttributesTest.java

```diff
@@ -0,0 +1,71 @@
+package com.eppo.sdk.dto;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class EppoAttributesTest {
+
+    @Test
+    void testSerializeEppoAttributesToJSONString() throws JSONException {
+        EppoAttributes eppoAttributes = new EppoAttributes(Map.of(
+          "boolean", EppoValue.valueOf(false),
+          "number", EppoValue.valueOf(1.234),
+          "string", EppoValue.valueOf("hello"),
+          "null", EppoValue.nullValue()
+        ));
+
+        String serializedJSONString = eppoAttributes.serializeToJSONString();
+        String expectedJson = "{ \"boolean\": false, \"number\": 1.234, \"string\": \"hello\", \"null\": null }";
+
+        JSONAssert.assertEquals(expectedJson, serializedJSONString, true);
```

Comment:
`JSONAssert()` ([link to library](https://github.com/skyscreamer/JSONassert)) is pretty slick. Tests for JSON equality so you don't have to worry about things like order.